scanner/cmd/scannerctl/main.go

```diff
@@ -18,6 +18,8 @@ import (
 )
 
 func main() {
+	serverName := flag.String("server-name", "", "Server name of the scanner service.")
+	port := flag.Int("port", 8443, "Port to use when connecting to the scanner service.")
```

Comment:
Usually, `<hostname>:<port>` is a single param called "address".

Reply:
My thought was to separate them so we can set `tlsOpts.ServerName = *serverName` but we could probably parse the hostname out from the address